### ISSUE TITLE
feat: Add or document inverse flag options

### ISSUE THREAD

**Issue by simsrw73** [walles/moar#124]
On Windows, using PowerShell Get-Help with $PAGER configured to use moar, it's cutting of lines when line numbers are displayed. I don't know if that's a bug in moar or PowerShell. PowerShell doesn't give any options for configuring paging at all. Line wrapping option doesn't help at all: words are still cut off. I can add the -no-linenumbers flag to the $MOAR variable to fix it, but then I ran in to the problem of how to show line numbers in other instances. It took me a long time before I blindly stumbled on the solution: -no-linenumbers=f. That's terribly ugly and not documented anywhere that I can see. 

Can we have inverted flags for boolean options? -linenumbers or -show-linenumbers? Or at the very least, document how to supply values to boolean flags? This was not obvious to me.

**Comment by walles** [comment]
Sounds like fixing that "it's cutting of lines when line numbers are displayed" would be good as well.

Could you please add to this ticket:
* A screenshot of the problem
* A `moar` version number

Since I don't have access to any Windows machine I may not be able to repro this, but at least having a screenshot would be good.

**Comment by walles** [comment]
Also, just out of curiosity, how did you install `moar` on Windows?

**Comment by simsrw73** [comment]
Installed with scoop package manager (https://scoop.sh)

❯ scoop info moar

Name        : moar
Description : A pager designed to work without any configuration
Version     : 1.11.4
Bucket      : extras
Website     : https://github.com/walles/moar
License     : BSD-2-Clause-Views
Updated at  : 2023-01-11 11:50:52
Updated by  : github-actions[bot]
Installed   : 1.11.4
Binaries    : moar.exe

$PAGER=moar
$MOAR='--style catppuccin-mocha --no-linenumbers'
![nolineno](https://user-images.githubusercontent.com/12434447/220741734-8229852a-da19-49ff-9017-84ae1f225a45.png)

$PAGER=moar
$MOAR='--style catppuccin-mocha'
![lineno](https://user-images.githubusercontent.com/12434447/220741815-b4e29f65-f601-4cbe-943c-7aad4d46af94.png)

Fumble fingers found another small issue. Hit the Break key instead of PrintScreen key
![break](https://user-images.githubusercontent.com/12434447/220745232-3ace375d-18ad-4579-9e65-ee4dcc7ac3d6.png)
It terminates moar and leaves the terminal in a somewhat broken state: trying to scroll the buffer with mouse 
just output escape sequences to the terminal. Sorry, for dumping all that here. Did you want me to break this in to multiple issues... ?

**Comment by simsrw73** [comment]
I can repro this on Linux. 

- Installed latest go
- Built moar
- Installed powershell (https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3)
- Run powershell just like running any other shell
```powershell
pwsh
update-help
help get-help
exit
```
You can setup the PAGER & MOAR vars before runing pwsh or from within
```powershell
$Env:PAGER = moar
$Env:MOAR = '-no-linenumbers'
```

**Comment by walles** [comment]
Is the problem that `pwsh` line breaks the help text before sending it to `moar`, just like `man`?

Ref: https://github.com/walles/moar/issues/119#issuecomment-1399616473

Otherwise, can you point out in detail one thing you'd like to look different in one of your screenshots?

**Comment by walles** [comment]
> Did you want me to break this in to multiple issues... ?

Yes :)

**Comment by simsrw73** [comment]
> Is the problem that `pwsh` line breaks the help text before sending it to `moar`, just like `man`?

Yes. I don't believe this is an issue with moar. That's why I didn't write that issue up specifically. The `help` built-in function of pwsh formats the help document according to the width of the terminal, not accounting for moar or any other utility adding line numbers or any extra columns. It's not even useful to have line numbers in help, it only came up because line numbers is moar's default, which would be great if pwsh gave a way to override the PAGER command specifically for help.

So, by necessity, what I need is to be able to set MOAR to default to -no-linenumbers, and then invoke moar with line numbers in cases where I need it (probably with an alias). That is the main issue I was intending to bring up here: Adding/documenting inverse flags. Eg. -show-linenumbers / -no-linenumbers / -no-linenumbers=f. I hate that last syntax's double negative, but it works as is right now. It's just not documented.

Sorry we got side-tracked.
